test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -232,7 +232,7 @@ public async Task GivenAnInvalidSearchParam_WhenCreatingParam_ThenMeaningfulErro
             }
         }
 
-        [SkippableFact]
+        [Fact(Skip = "To re-enable when the bug(82891) is fixed")]
```

Comment:
As discussed with @feordin I am skipping couple of tests which is failing due to bug. Once that gets resolved, we will re-enable the tests. Workitem created for this one - https://microsofthealth.visualstudio.com/Health/_boards/board/t/Resolute/Stories/?workitem=82891